tutorials/distributed-keras-on-cmle/index.md

```diff
@@ -0,0 +1,37 @@
+* [Google Cloud Machine Learning Engine (CMLE)](https://cloud.google.com/ml-engine/) - CMLE is a managed service that enables you to easily build machine learning models that work on any type of data, of any size.  It facilitates two activities: One, decreasing training time by allowing you train models using large numbers of CPUs, GPUs and TPUs, and two,
+allowing you to train models in parallel.  Both of these activities further enable you to iterate faster with less overhead.
+
+## Prerequisites
+
+* Run through the [Cloud Datalab quickstart](https://cloud.google.com/datalab/docs/quickstarts):
+  * Create a project on Google Cloud Platform.
+  * Ensure Compute Engine (GCE) and CMLE APIs are enabled.
+  * Create an instance of Cloud Datalab.
+* Create a new notebook within Datalab and execute the following from a code cell, `!gsutil cp gs://gcp-community/tutorials/distributed-keras-on-cmle/*.ipynb .`
+* Open `1-start-here.ipynb` and enjoy.
+
+## Activities
+
+* [You'll learn (or revise) creating custom estimators in tf.contrib.learn](https://github.com/GoogleCloudPlatform/community/blob/master/tutorials/distributed-keras-on-cmle/2-original-neural-net.ipynb) from [this tutorial](https://www.tensorflow.org/extend/estimators) by running the [original code](https://github.com/tensorflow/tensorflow/blob/r1.2/tensorflow/examples/tutorials/estimators/abalone.py).
+* [You'll run a barebones version of the same model that will serve as a basis for migrating the model to Keras](https://github.com/GoogleCloudPlatform/community/blob/master/tutorials/distributed-keras-on-cmle/3-tweaked-original-neural-net.ipynb).
+* [You'll run a version of the same model implemented using Keras](https://github.com/GoogleCloudPlatform/community/blob/master/tutorials/distributed-keras-on-cmle/4-keras.ipynb) for the dataflow graph definition and tf.contrib.learn for other functionality such as the loss function and utilites such as the training/evaluation loop.
+* [You'll run a production-grade version of the Keras model locally](https://github.com/GoogleCloudPlatform/community/blob/master/tutorials/distributed-keras-on-cmle/5-keras-full.ipynb) that adds in the code required to train the model on CMLE.
+* [You'll use CMLE via the gcloud CLI to execute the same model locally and remotely](https://github.com/GoogleCloudPlatform/community/blob/master/tutorials/distributed-keras-on-cmle/6-distributed-keras.ipynb) on Google Cloud Platform (GCP).
+
+The remainder of this notebook includes a number of housekeeping items required to run later notebooks (upgrading Tensorflow to 1.2 and downloading training/test datasets).
```

Comment:
Which notebook? This is a markdown file. I see that this text is repeated in the first notebook, where this line makes more sense. I suggest removing it here, where it is potentially confusing.

Also, any changes here should be reflected in the notebook version.